newrelic.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['AirGB - AboutService'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: '874e0832e53d48fb3cd5cf76151f28d42f3ccf19',
```

Comment:
maybe add newrelic.js to gitignore to keep key private

Reply:
Will implement on next commit. Thanks Servio!